.github/workflows/build-pr-image.yml

```diff
@@ -0,0 +1,97 @@
+
+name: Build PR Docker Image
+
+on:
+  push:
+    branches:
+      - "main"
```

Comment:
This will cause the image be build when merged to main. Inst that already covered by `openobserve-dev:latest-amd64` ?